src/Carousel.js

```diff
@@ -301,4 +303,8 @@ const Carousel = React.createClass({
   }
 });
 
+Carousel = Object.assign(Carousel, { Caption });
+
 export default Carousel;
+
+export { Caption };
```

Comment:
this amounts to a breaking change, so i'd either need to go in the 0.29 branch or (preferablly) just remove the named export since the Static property on Carousel is fine